Documentation/gettingstarted/istio.rst

```diff
@@ -342,7 +383,7 @@ Apply this route rule:
 .. parsed-literal::
 
     $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-istio/route-rule-reviews-v1-v2.yaml
-    routerule "reviews-default" configured
+    routerule.config.istio.io "reviews-default" configured
 
 Check in your web browser that stars are appearing in the Book Reviews
 roughly 50% of the time.  This may require refreshing the page for a
```

Comment:
I'm not seeing that.

Reply:
I fixed that by updating to the latest v1alpha3 API.